author.rb

```diff
@@ -0,0 +1,18 @@
+# Create an Author class with an association to the Item class.
+require './classes/item'
+class Author < Item
+  attr_accessor :id, :first_name, :last_name, :items
+
+  def initialize(id, first_name, last_name, items)
+    super(first_name, last_name)
```

Comment:
If you see when you call super (first_name,last_name) those are not properties of super (.ie the Item class), so you can delete this line.

Reply:
Thanks, I will